Jellyfin.Api/Controllers/BrandingController.cs

```diff
@@ -0,0 +1,46 @@
+using Jellyfin.Api.Models.Branding;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller.Configuration;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// The branding controller.
+    /// </summary>
+    public class BrandingController : BaseJellyfinApiController
+    {
+        private IServerConfigurationManager _serverConfigurationManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BrandingController"/> class.
+        /// </summary>
+        /// <param name="serverConfigurationManager">Instance of ServerConfigurationManager.</param>
+        public BrandingController(IServerConfigurationManager serverConfigurationManager)
+        {
+            this._serverConfigurationManager = serverConfigurationManager;
+        }
+
+        /// <summary>
+        /// Endpoint for getting a servers branding settings.
+        /// </summary>
+        /// <returns>Branding settings of the server.</returns>
+        [HttpGet("Configuration")]
+        public BrandingDto GetBrandingOptions()
+        {
+            return this._serverConfigurationManager.GetConfiguration<BrandingDto>("branding");
```

Comment:
Per #2876 and its created DTO-classes i gained the impression that the goal would be to be somewhat more independent from MediaBrowser-Assemblies (```NotificationDto``` vs. ```Emby.Notifications.Notification```) but maybe i missed a point there. Im happy to delete the BrandingDto if i get the memo about that

Reply:
That's an interesting point, I'm not sure if we made a decision on things like that. API objects can sometimes differ from their respective internal objects, but I also dislike code duplication. I think the original WebAPI port (the startup wizard) was creating new objects, but I'm not sure.

Reply:
The DTO class in #2876 wasn't newly created, it already existed in the old notification service, just with a different name `Notification`.

Regardless of whether we want to create a new object or not, the DTO type should not be passed to `GetConfiguration<>` directly. We should call `GetConfiguration<BrandingOptions>` and then the returned `BrandingOptions` object should be converted into a `BrandingOptionsDTO` instance.

In other projects I have always used DTO separate from the internal objects as a rule, and used [AutoMapper](https://automapper.org/) to easily map between the DTO and internal representation. But I also don't like code duplication and I know that pattern is often criticized. So I'm not opposed to using the internal object directly and only using DTO-type objects in endpoints where it is actually necessary.

Lastly, I think logic like this should all go into a separate service-layer class, while the controller only handles HTTP-specific logic like writing to the response body, setting headers, etc. `_serverConfigurationManager` is playing the role of the service layer here, but normally the mapping between internal object and DTOs would also be in the service layer, so that should be considered if we decide to use a DTO.

Reply:
I will prepare the removal of the DTO-class to get this PR in line with the other Migration PRs, i guess. I didnt mean to make my PR the place for these fundamental thoughts (event tho they should be considered in future)